wd_helper.au3

```diff
@@ -2180,6 +2180,52 @@ Func _WD_JsonActionPause($iDuration)
 	Return $sJSON
 EndFunc   ;==>_WD_JsonActionPause
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_JsonCookie
+; Syntax ........: _WD_JsonCookie($sName, $sValue[, $sPath = Default[, $sDomain = Default[, $bSecure = Default [,
+;                  $bHTTPOnly = Default[, $iExpiryTime = Default[, $sSameSite = Default]]]]]])
+; Parameters ....: $sName               - The name of the cookie.
+;                  $sValue              - The cookie value.
+;                  $sPath               - [optional] This defines the cookie path. Default is '/'.
+;                  $sDomain             - [optional] This defines the domain the cookie is visible to. Default is '' - will be ommited and current browsing context’s of active document’s will be used.
+;                  $bSecure             - [optional] This defines whether the cookie is a secure cookie. Default is False.
+;                  $bHTTPOnly           - [optional] This defines whether the cookie is an HTTP only cookie. Default is False.
+;                  $iExpiryTime         - [optional] This defines when the cookie expires, specified in seconds since Unix Epoch. Default is 0 - will be ommited.
+;                  $sSameSite           - [optional] This defines whether the cookie applies to a SameSite policy. Default is '' - will be ommited and this mean "None". One of the following modes can be used:
+;                  |None
+;                  |Lax
+;                  |Strict
```

Comment:
Please remove the "Default is" portions as the function no longer supplied default values. Same thing with the "will be omitted" portions.